Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/configadmin/ConfigAdminUnsecured.java

```diff
@@ -62,28 +65,32 @@
  */
 public class ConfigAdminUnsecured implements gov.hhs.fha.nhinc.configadmin.EntityConfigAdminPortType {
 
-    /**
-     *
-     */
-    private static final String UPDATE_CERTIFICATE_SUCCESS = "Update certificate Success";
+    private static final String UPDATE_CERTIFICATE_SUCCESS = "Update certificate success";
+    private static final String INVALID_USER = "Invalid user credentials";
```

Comment:
It is more about mismatch token.  Maybe the error should be "Mismatch Hash Token'